ow_system_plugins/base/controllers/billing.php

```diff
@@ -31,6 +31,7 @@
  */
 class BASE_CTRL_Billing extends OW_ActionController
 {
+    use BASE_CTRL_BillingMethods;
 
```

Comment:
Change the name of BASE_CTRL_BillingMethods with BASE_CLASS_BillingMethodsTrait;